packages/kbn-es/src/serverless_resources/project_roles/security/roles.yml

```diff
@@ -307,13 +307,18 @@ threat_intelligence_analyst:
         - logstash-*
       privileges:
         - read
+    - names:
+        - .asset-criticality.asset-criticality-*
+      privileges:
+        - read
+        - write
     - names:
         - .lists*
         - .items*
-        - .asset-criticality.asset-criticality-*
       privileges:
         - read
         - write
+        - view_index_metadata
```

Comment:
<img width="2558" alt="Screenshot 2024-05-31 at 17 44 47" src="https://github.com/elastic/kibana/assets/92328789/e240b49f-6be4-4b0b-9a17-56205cada12e">

Role lacks `manage` privilege,  which does not allow to create value list data stream and subsequently upload list. Is this expected behaviour? Do we have documented expectations for each role?

Reply:
When I first tested this I had to first visit the detections page with something like a platform_engineer role. Didn't have to upload a list just visited the page. And then logged back in with the `threat_intelligence_analyst` / `endpoint_policy_manager` users and was able to upload a list. I'm assuming someone with these two roles will not be the initial users to initiate the setup for the detections page?

Reply:
I wanted to give those users the minimum privileges necessary to do the tasks but if product is okay with handing over `manage` privileges to those roles then that's fine too. I will ask on the github issue.

Reply:
Yes, same for me, I was able to upload lists, after admin created underlying data stream. 

I am curious what exact expectation for these roles. If admin has to visit page first, it's fine, if it's by design